src/api/mod.rs

```diff
@@ -246,4 +406,108 @@ mod api_client_tests {
 
         assert!(!result.is_empty())
     }
+
+    // Manually refresh token three times
+    #[tokio::test]
+    async fn test_refresh_token() {
+        let api = get_api().await;
+
+        let lock = api.body.token.read().await;
+        let initial_token = lock.token.clone();
+        drop(lock);
+
+        // Sleeps added here to avoid getting same token
+        // because it runs too fast (maybe it's a bug?)
```

Comment:
This is not a bug, the same refresh token is always returned when calling /refresh.

Reply:
it's actually returning same token not refresh token

Reply:
@myssto Mind investigating?

Reply:
Fixed here:
- [ ] https://github.com/osu-tournament-rating/otr-api/pull/186